.github/workflows/release.yml

```diff
@@ -0,0 +1,98 @@
+name: release
+
+on:
+  workflow_dispatch:
+    inputs:
+      qemu_repo:
+        description: 'QEMU repo (e.g. https://github.com/qemu/qemu)'
+        required: false
+      qemu_version:
+        description: 'QEMU version (e.g. v5.2.0)'
+        required: false
+      target:
+        description: 'Bake target'
+        required: true
+        default: 'mainline'
+      latest:
+        description: 'Create latest tag'
+        required: true
+        default: 'false'
+      dry-run:
+        description: 'Dry run'
+        required: false
+        default: 'true'
+
+env:
+  REPO_SLUG: tonistiigi/binfmt
+
+jobs:
+  release:
+    runs-on: ubuntu-latest
+    steps:
+      -
+        name: Checkout
+        uses: actions/checkout@v2
+      -
+        name: Prepare
+        run: |
+          QEMU_REPO=$(docker buildx bake --print ${{ github.event.inputs.target }} | jq --raw-output '.. | .args?.QEMU_REPO | select(.)')
```

Comment:
why do you parse out the values form bake? instead of just setting the value with a condition like you do already.

Reply:
I think it's more appropriate to use defaults from the bake def if inputs are empty. This allows us not to duplicate these values in the workflow and also to extract the right values according to the selected target.

Reply:
I don't think this is pattern we should be suggesting to any bake users. The logic is that defaults are defined in file and you can override them if needed. Not that you need to use jq to parse out the values from the file and then substitute them with your own values if needed.

Reply:
I completely agree with that but here we have a rough case where we need to create a GitHub Release in the workflow with its git tag (which is `$QEMU_VERSION-$GITHUB_RUN_NUMBER`) so I have to grab default QEMU_VERSION from bake definition if none is defined as input in the workflow dispatch event.